dev/com.ibm.ws.rest.handler.config_fat/fat/src/com/ibm/ws/rest/handler/config/fat/audit/ConfigRESTHandlerAuditTest.java

```diff
@@ -0,0 +1,136 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.rest.handler.config.fat;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.log.Log;
+import com.ibm.websphere.security.audit.AuditEvent;
+import com.ibm.ws.rest.handler.config.fat.FATSuite;
+import com.ibm.websphere.security.audit.AuditConstants;
+import com.ibm.ws.security.audit.fat.common.tooling.AuditAsserts;
+import com.ibm.ws.security.audit.fat.common.tooling.AuditMessageConstants;
+import com.ibm.ws.security.audit.fat.common.tooling.RecentAuditFileStream;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.EERepeatActions;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import componenttest.topology.utils.HttpsRequest;
+import componenttest.rules.repeater.FeatureReplacementAction;
+import componenttest.rules.repeater.EmptyAction;
+
+import com.ibm.ws.security.audit.fat.common.tooling.AuditCommonTest;
+
+@RunWith(FATRunner.class)
+public class ConfigRESTHandlerAuditTest extends FATServletClient {
+    private static final Class<?> c = ConfigRESTHandlerAuditTest.class;
+    public static final String REST_EVENT = AuditEvent.EVENTNAME + "=" + AuditConstants.SECURITY_REST_HANDLER_AUTHZ;
+    protected static String auditFileLogPath = null;
+    protected static Boolean auditLogFilePathExists = false;
+
+    protected String auditLogFilePath = null;
+    public static final String DEFAULT_AUDIT_LOG = "audit.log";
+
+    @ClassRule
+    public static RepeatTests r = EERepeatActions.repeat("com.ibm.ws.rest.handler.config.audit.fat",
+    EERepeatActions.EE10, // EE10
+    EERepeatActions.EE9, // EE9
+    EERepeatActions.EE8).with(new EmptyAction().fullFATOnly()).andWith(new FeatureReplacementAction(AuditCommonTest.AUDIT_1_0_FEATURES, AuditCommonTest.AUDIT_2_0_FEATURES));
```

Comment:
I am guessing this doesn't give you what you want.  EE 10 will run with audit-1.0 and 2.0 in lite mode and EE 7 will run with audit 1.0 and 2.0 in full mode I believe.  Otherwise everything else will run just with audit 1.0.

Reply:
Thanks for catching that, I was debugging that issue.